src/backend/database/resume/getResume.ts

```diff
@@ -0,0 +1,11 @@
+import { collections } from "../constants";
+import { database } from "../../config";
+import { get, ref } from "firebase/database";
+
+export const getResume = async (cvId: string) => {
+  return await get(ref(database, collections.resumes + cvId))
+    .then((data) => {
+      return data.val();
```

Comment:
There is no data, there is a snapshot. If snapshot.exist() then return data.val(), else return null.